apps/schematic/api/schematic_api/test/test_synapse_endpoints.py

```diff
@@ -140,7 +140,7 @@ def test_submit_manifest_csv(self) -> None:
         )
         body = csv_to_bytes(CORRECT_MANIFEST_PATH)
         response = self.client.open(url, method="POST", headers=HEADERS, data=body)
-        self.assert500(response, f"Response body is : {response.data.decode('utf-8')}")
+        self.assert200(response, f"Response body is : {response.data.decode('utf-8')}")
```

Comment:
why this gets changed to `assert200`? is it because of a change in recent schematic?

Reply:
oh i see. that was a typo before?